content/en/llm_observability/instrumentation/otel_instrumentation.md

```diff
@@ -66,6 +66,32 @@ After your application starts sending data, the traces automatically appear in t
 
 </div>
 
+## Disabling LLM Observability conversion
+
+If you'd only like your generative AI spans to remain in APM and not appear in LLM Observability, you can disable the automatic conversion by setting the `dd_llmobs_enabled` attribute to `false`. Setting this attribute on any span in a trace prevents the entire trace from being converted to LLM Observability.
```

Comment:
we should move all this content here to the new doc that'll be added to docs.datadoghq.com/opentelemetry/, linking out to that new section/doc from this section. These instructions are more relevant for existing APM customers

Reply:
Yeah, none of the other integrations get their own page, this includes completely separate products, instead, they link out to their features.